src/cli/build.rs

```diff
@@ -15,13 +38,8 @@ fn main() {
 
     // Fetch OS information if on unix
     if cfg!(unix) {
```

Comment:
make variable name "os_name" better fit here.   It is just more OS name, but OS details.